distribution/docs/src/main/resources/content/_installing/firewall-config.adoc

```diff
@@ -37,8 +37,13 @@ Firewalls will need to be configured to open these ports in order for external s
 |44444
 |RMI Server Port
 
+|8994
```

Comment:
is this port only available to localhost loopback or can it be hit by external instances also?

Reply:
scratch that i read on and see that its only available for localhost can you say that here also